node-w3capi/api.js

```diff
@@ -46,5 +44,15 @@ export async function fetchData(groupId) {
     });
   }
 
-  return { users, orgs: Object.values(orgsMap) }
+  const orgs = Object.values(orgsMap)
+
+  // sort by org name
+  orgs.sort((a, b) => a.name.localeCompare(b.name, "en", { ignorePunctuation: true }))
+
+  // sort users by w3c id
+  orgs.forEach(organization => {
+    organization.orgUsers.sort((a, b) => a.id - b.id)
+  })
+
+  return { users, orgs }
```

Comment:
@csarven sorting happens here, while the data in your comment somehow didn't get sorted